ui/src/app/workflows/components/workflows-summary-container/workflows-summary-container.scss

```diff
@@ -0,0 +1,45 @@
+@import 'node_modules/argo-ui/src/styles/config';
+
+.wf-summary-container {
```

Comment:
can we avoid using new styles?

Reply:
Are you suggesting using inline styles?

Reply:
I’m suggesting no new styles.

Reply:
Removing the new styles and adding the available `white-box` class to the outer div looks like this.

<img width="1790" alt="Screenshot 2022-09-30 at 7 20 50 AM" src="https://user-images.githubusercontent.com/5287720/193173020-7bcdb9b9-461e-4f4a-b1e6-8b5818b2e7ea.png">

I think some inline styling is needed if no new styles are allowed.